src/dialog/dialog.test.tsx

```diff
@@ -1,6 +1,5 @@
-import * as React from 'react';
-
 import { cleanup, fireEvent, render } from '@config/testing';
+import * as React from 'react';
```

Comment:
Do these changes have an intention?

Reply:
auto-formatting i have reverted the changes